src/internal/pfsdb/pfsdb.go

```diff
@@ -68,9 +67,20 @@ var CommitsRepoIndex = &col.Index{
 	},
 }
 
-var commitsIndexes = []*col.Index{CommitsRepoIndex}
+var CommitsBranchlessIndex = &col.Index{
+	Name: "branchless",
+	Extract: func(val proto.Message) string {
+		return CommitBranchlessKey(val.(*pfs.CommitInfo).Commit)
+	},
+}
```

Comment:
What is this indexing? My understanding based on the name is that it's commits that aren't on any branches, but it doesn't seem like there should be any commits like that so I feel like I'm misunderstanding something.

Reply:
It's an index on the commit without specifying the branch, I'm open to renaming this (it isn't used very often, basically just when resolving a commit from a user-provided `pfs.Commit` that doesn't specify a branch).  Maybe `CommitAnyBranchKey`?  Open to suggestions.